src/assets/videos/adHocVideos.ts

```diff
@@ -28,4 +28,4 @@ export const adHocVideos: IVideoInfo[] = [
     genres: ['documentary'],
     roles: ['director_of_photography', 'director_producer'],
   },
-]
+] as const satisfies Readonly<IVideoInfo[]>
```

Comment:
`satisfies` gives us to ability to type check while maintaining the `const` values inside each object